File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/EtteroppgjorGrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.beregning2011.PensjonUnderUtbetaling;

import java.io.Serializable;

/**
 * @author Tatyana Lochehina PK-11673
 * @author Swiddy de Louw PK-16620
 */
public class EtteroppgjorGrunnlag implements Serializable {

    

    /**
     * Ytterligere informasjon om etteroppgjør
     */
    private UforetrygdEtteroppgjor uforetrygdEtteroppgjor;

    /**
     * Den PensjonUnderUtbetaling som blir revurdert.
     */
    private PensjonUnderUtbetaling pensjonUnderUtbetalingForRevurdering;


}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/EtteroppgjorGrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.beregning2011.PensjonUnderUtbetaling
import java.io.Serializable

/**
 * @author Tatyana Lochehina PK-11673
 * @author Swiddy de Louw PK-16620
 */
class EtteroppgjorGrunnlag : Serializable {
    /**
     * Ytterligere informasjon om etteroppgjør
     */
    var uforetrygdEtteroppgjor: UforetrygdEtteroppgjor? = null
    /**
     * @return the pensjonUnderUtbetalingForRevurdering
     */
    /**
     * @param pensjonUnderUtbetalingForRevurdering the pensjonUnderUtbetalingForRevurdering to set
     */
    /**
     * Den PensjonUnderUtbetaling som blir revurdert.
     */
    var pensjonUnderUtbetalingForRevurdering: PensjonUnderUtbetaling? = null

}